frontend/src/components/RequestedFurnishings.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from "react";
+import styles from "src/components/RequestedFurnishings.module.css";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Typography from "@mui/material/Typography";
+import { SingleDetail, ListDetail } from "@/components";
+import { useParams } from "next/navigation";
+import { getVSR, type VSR } from "@/api/VSRs";
+
+export const RequestedFurnishings = () => {
+  const [vsr, setVSR] = useState<VSR>({} as VSR);
+  const { id } = useParams();
+
+  useEffect(() => {
+    getVSR(id as string).then((result) => {
+      if (result.success) {
+        setVSR(result.data);
+      }
```

Comment:
Also, in the one place we call `getVSR`, we should have some error handling. If the backend returns an error, we should display a corresponding error message to the user (e.g. network error, VSR not found). The way it is now, we still display the same VSR page but with none of the fields filled in, so it's hard to tell there's an error

Reply:
I added an error handling, so an error will be displayed where the name would be. Would we awnt a modal in the future?
![image](https://github.com/TritonSE/PAP-Inventory-Processing/assets/25836557/93844607-f55b-4b4e-9d7d-bb464c6043dd)

Reply:
Ok, this is good for now. We might want a different way of displaying errors in the future, we can talk to the designers. But can you change it so if there's an error, we only display the error message and don't display the other VSR components?